### PR TITLE
N53: add godot-rapier

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #1547_

### CODE REVIEW

**Comment by janhohenheim** [comment]
Ping me when you're ready for a review :)

**Comment by Vrixyz** [comment]
@janhohenheim I did the last few edits I wanted :)

File: content/news/053/index.md [hunk 1]
```diff
@@ -121,6 +121,33 @@ _Check out the [web examples][chuot-examples]!_
...
 
+### Godot Rapier: Rewrite it in Rust
+
+{{ video_figure(
+    type="video/mp4",
+    alt="Godot + Rapier logo make up the 'Godot Rapier' logo ; a few balls fall on it, showing off soft physics.",
+    src="godot_rapier.mp4",
+    caption="Godot Rapier showcase") }}
+
+[Godot Rapier][godot_rapier_github] is a Physics library for [Godot][godot], using [Rapier][rapier]:
```
Comment (Ughuuu) [inline content/news/053/index.md:132]: [Godot Rapier][godot_rapier_github] written by [@Ughuuu](https://github.com/Ughuuu) is a ..

Comment (Ughuuu) [inline content/news/053/index.md:132]: is a Physics Plugin for Godot

File: content/news/053/index.md [hunk 2]
```diff
@@ -121,6 +121,33 @@ _Check out the [web examples][chuot-examples]!_
...
+a physics engine written in Rust.
+
+Godot Rapier was written in C++, and is in the final "beta" stage of
+being rewritten completely in Rust.
+
+Its author wrote [an article on
+Reddit][godot_rapier_reddit] with more details. Since then,
+the web exports have been fixed.
```
Comment (Ughuuu) [inline content/news/053/index.md:140]: Read more about this on the `[Rapier Godot Progress](https://godot.rapier.rs/docs/progress)` page.

**Review by Ughuuu — APPROVED** [review]

**Review by janhohenheim — APPROVED** [review]

**Comment by janhohenheim** [comment]
Did a little editing pass. Thanks for the PR :)

**Pull request merged**